mcclim.asd

```diff
@@ -72,7 +72,8 @@ interface management system."
 (defsystem #:mcclim/extensions
   :depends-on (#:mcclim-bitmaps
                #:conditional-commands
-               #:mcclim-layouts/tab))
+               #:mcclim-layouts/tab
+               #:mcclim-bezier))
 
```

Comment:
I'm glad we have crafted it as an extension. Are you planning to write documentation for it? If not, I'll add issue assigned to myself and make one.